sdk/core/Cargo.toml

```diff
@@ -54,3 +54,4 @@ enable_reqwest_rustls = ["reqwest/rustls-tls"]
 test_e2e = []
 azurite_workaround = []
 xml = ["quick-xml"]
+wasm-bindgen = ["time/wasm-bindgen"]
```

Comment:
My original thinking was that the `wasm32` target was not necessarily tied to use of `wasm-bindgen` (e.g. if you want to use `wasi` rather than run in a browser).  However, for now this will be the case (`wasi` support seems quite a long way off..), so I've changed to automatically set the `time` crate `wasm-bindgen` feature on `wasm32` builds.  Unfortunately this is a little ugly - have to specify the time dependency twice with different feature sets (I couldn't find a better way to do it).

Reply:
What are your thoughts on the following?  Due to features being additive, you could leave the original `time` definition, then use this:
```toml
[target.'cfg(target_arch = "wasm32")'.dependencies]
time = { version = "0.3.10", features = ["wasm-bindgen"] }
```